gitlab4j-models/src/main/java/org/gitlab4j/api/models/ContainerExpirationPolicy.java

```diff
@@ -4,15 +4,26 @@
 
 import org.gitlab4j.models.utils.JacksonJson;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class ContainerExpirationPolicy implements Serializable {
     private static final long serialVersionUID = 1L;
 
     private String cadence;
     private Boolean enabled;
+
+    @JsonProperty("keep_n")
```

Comment:
Currently the way the models are written, we are not using `@JsonProperty`. There was an attempt in https://github.com/gitlab4j/gitlab4j-api/pull/1198, but it never was finished.

For consistency reason, I would not add this now.